OpenStatesApi/CommitteeMember.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace OpenStatesApi
+{
+    public class CommitteeMember
+    {
+        [JsonProperty( "leg_id" )]
+        public string LegislatorId { get; set; }
+        public string Name { get; set; }
+        public string Role { get; set; }
```

Comment:
I tried utilizing and extending the existing RoleType enum but consistently got errors when testing or attempting to call the method. Pennsylvania has a concept of minority chair, which is really just "[Party] Chair" in the Sunlight data. However it kept saying it couldn't find it. Creating a new enum of CommitteeRoleType did not correct the issue (and seemed to be a duplication of work, anyway, since it matched RoleType in all but the enumerations inside the switch).